### DISCUSSION TITLE
Lost Access to GitHub Copilot for Student

### DISCUSSION THREAD

**Discussion by arthursdo** [community/community#58552] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hello everyone,

I hope this message finds you well. I'm seeking your help and insights regarding an issue I've been encountering with GitHub Copilot.

I am a student and have been using the GitHub Student Developer Pack, which includes free access to GitHub Copilot. Recently, however, I have lost my access to GitHub Copilot and I'm uncertain why.

I've ensured my student status is updated and verified. I've submitted my recent student credentials and received confirmation that they were successfully verified. However, when I attempt to activate GitHub Copilot from the student pack page, I'm prompted to pay $10.

Moreover, the message indicates that I am not eligible for the 30-day free trial, which is puzzling considering my student status and the benefits the student pack is supposed to offer.

I'm hoping that someone in this community has experienced a similar issue and can provide some guidance on how to resolve this. I've relied heavily on GitHub Copilot for my studies and it's been instrumental in my learning process.

Any advice or direction would be deeply appreciated. Thank you in advance for your time and help.

**Comment by davecheney** [comment, marked as answer]
@arthursdo your 2021 student coupon expired on May 16, 2023. If you are still a student you can re-apply for a student developer pack which will include a 2023 student coupon. Coupons are valid for 2 years from the date of application.
